tox.ini

```diff
@@ -19,3 +19,11 @@ commands =
 setenv =
   ANSIBLE_CALLBACK_PLUGINS = {envsitepackagesdir}/ara/plugins/callbacks
 commands = {posargs}
+
+[testenv:pytest]
+deps =
+  ansible
```

Comment:
nit, maybe change this to ansible-core, to minimize the install. But unsure what deps really look like

Reply:
Oh +1, I always forget about that.